### ISSUE TITLE
Support non North American Regions (Invoke-ProviderList PowerPlatform error)

### ISSUE THREAD

**Issue by albandrod** [cisagov/ScubaGear#201]
I'm executing scuba as:

$Splat = @{
    CertificateThumbPrint = $thumbprint;
    AppID = $clientId;
    Organization = $tenantName;
    ProductNames = "*";
}
Invoke-Scuba @Splat -OutPath $client

but I'm receiving the following error:

Invoke-ProviderList : Error with the PowerPlatform Provider. See the exception message for more details:  "Power        Platform Provider ERROR: The M365Environment parameter value is not set correctly which WILL cause the Power Platform   report to display incorrect values.  

I tried to put into de Splat the variable M365Environment = "commercial", but it still shows the same error message

**Comment by gdasher** [comment]
When that error prints, it should print the environments in question after the break ("----"). 
 See $EnvErrorMessage [here]( https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/Modules/Providers/ExportPowerPlatformProvider.psm1).

Did you get that full output or was it truncated?

If it was truncated, can you also provide the json you get (specifically tenant_region_scope and tenant_region_subscope) from this url (adjust TLD if using gcchigh/dod):

`https://login.microsoftonline.com/mytenantname.onmicrosoft.com/.well-known/openid-configuration`

Basically what this check that you are hitting is doing is confirming that the environment type the tenant reports lines up with the one provided because we saw bad data (not just errors) with powerplatform when they did not.
Its possible there is a bug in our logic that we hadn't considered.  It looks like we are assuming north america, so I can see a potential problem already, that should be easy to fix.
